tests/components/github/test_init.py

```diff
@@ -1,7 +1,7 @@
 """Test the GitHub init file."""
 from pytest import LogCaptureFixture
 
-from homeassistant.components.github.const import CONF_REPOSITORIES
+from homeassistant.components.github import CONF_REPOSITORIES
```

Comment:
Unrelated, pylint complained about it.

Reply:
Curious what the complaint was?

Reply:
https://github.com/home-assistant/core/blob/dev/pylint/plugins/hass_imports.py#L314